index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ro">
+    <meta charset="UTF-8">
+    
+    <head class="header">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> Save the planet</title>
+        <link href="./CSS/homestyle.css" rel=stylesheet>         
+    </head>
+    <body>
+        <section class="coverContainer">
```

Comment:
na bun, aici iti incepe tie practic tot hero-ul, toate clasele de css se scriu dupa BEM (mai citeste odata documentatia si intreaba-ma daca nu ai inteles). 

```suggestion
        <section class="hero">
```

mergi mai departe cu copiii: hero__container, hero__title etc.